Headers/CMN_FJX_MOD.F90

```diff
@@ -31,7 +31,7 @@ MODULE CMN_FJX_MOD
   ! Parameters
   !-----------------------------------------------------------------------
 
-  INTEGER, PARAMETER :: JVN_ = 166     ! Max number of J-values
+  INTEGER, PARAMETER :: JVN_ = 167     ! Max number of J-values
```

Comment:
This will only apply when running with FAST-JX, which is non-default except for the mercury simulation. Currently the max number of J-values is hard-coded in Cloud-J. Restructuring the memory is on the todo list so that we can instead pass it from the parent model.

Could you confirm that yours tests used FAST-JX rather than Cloud-J? If using Cloud-J then this line needs to change: https://github.com/geoschem/Cloud-J/blob/03313bb6acde16a9962c3a5e18a3f78b4d220faa/src/Core/cldj_cmn_mod.F90#L47

Reply:
This update is no longer needed because of a feature I added to Cloud-J where JVN_ is truly the max # of J-values rather than the exact number of entries in FJX_j2j.dat. I updated the value of 200 to be above what we actually need.